daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dfdlx.xsd

```diff
@@ -47,6 +48,27 @@
     </xs:restriction>
   </xs:simpleType>
 
+
+  <xs:attribute name="alignmentViaSkip" type="dfdl:YesNoEnum" default="no">
```

Comment:
Thoughts on changing this to something like "alignmentPolicy"? I've mentioned other tweaks to deal with the alignment optimizations, but that are more complicated to implement. It might be nice to have a single property that essentially lets us choose between different alginment optimizations. And they could vary from a very pessimistic algorithm like we have now, or this optimistic one that completely disables alignment like this, or somewhere in between? We'd only have two values for now, so funtionally the same as a yes/no choice, but we could expand it in the future.